### PR TITLE
Add typescript types + replace `var` + fix Appveyor CI

### PR DESCRIPTION
**Pull request by aminya**
- Added TypeScript types
- Exported askJupyter
- Converted `var` to `let` and `const` using `jscodeshift`
- Fixed Appveyor CI

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 language: node_js
 node_js:
-  - "5.4"
-  - "4.2"
+  - "12"
```
Comment (captainsafia) [inline .travis.yml:3]: Let's also add Node v14 here.

File: appveyor.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ environment:
         - PYTHON: "C:\\Python35-x64"
 
 install:
-    - ps: Install-Product node 6
+    - ps: Install-Product node 12
```
Comment (captainsafia) [inline appveyor.yml:9]: Unrelated: at some point, we should probably replace Travis + AppVeyor here with GitHub Actions.

File: .travis.yml (hunk 1 above)
Comment (aminya) [reply .travis.yml:3]:
```suggestion
  - "12"
  - "14"
```

**Review by captainsafia — APPROVED** [review]

**Comment by captainsafia** [comment]
Functionality is fine but it looks like some of the Travis tests are wonky.

**Comment by aminya** [comment]
Yeah, however the changes in this PR are irrelevant to that. Not sure how to fix Travis.

**Comment by rgbkrk** [comment]
Thanks for these fixes!

**Comment by aminya** [comment]
You're welcome! Registering a new version would be appreciated.

**Pull request merged**